lib/api/controllers/securityController.js

```diff
@@ -329,6 +330,9 @@ module.exports = function SecurityController (kuzzle) {
         if (newRequestObject.data._id !== undefined) {
           pojoUser._id = newRequestObject.data._id;
         }
+        else {
+          pojoUser._id = uuid.v4();
```

Comment:
Why don't we put `'anonymous'` instead, so that the user repository attributes an anonymous profile?
That would be more efficient than reading the cache, and then the database, to search for a purposely unknown user.